Sources/Build/LLBuildManifestBuilder.swift

```diff
@@ -53,6 +53,9 @@ public class LLBuildManifestBuilder {
     var buildParameters: BuildParameters { self.plan.buildParameters }
     var buildEnvironment: BuildEnvironment { self.buildParameters.buildEnvironment }
 
+    /// Mapping from Swift compiler path to Swift get version files.
+    var swiftGetVersionFiles = [AbsolutePath: AbsolutePath]()
```

Comment:
do we need to worry about concurrency here?

Reply:
nope there is no currency in play with manifest builder